core/model-api/src/main/java/org/eclipse/rdf4j/model/base/AbstractBNode.java

```diff
@@ -22,6 +20,8 @@
 
 	private static final long serialVersionUID = -437354568418943981L;
 
+	////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
This is somewhat minor (so don't feel the need to immediately take these out again), but: open question (to @hmottestad and @barthanssens as well) - what's your opinion on the usefulness of this kind of "visual divider" commenting? I've been taking most of the legacy marker comment blocks ("constructors" / "public methods") etc out because they tend to get malformed and also just wrong over time (people putting private methods in the public methods bit etc). 

I'm personally not in favor of adding this kind of thing at all. But if we feel there's value in organising code this way then I'd like a consistent style adopted, at least.

Reply:
When working with a folding code editor on a complex class, I find it useful to be able to visually group related methods.

This particular graphic solution is probably too imposing: my personal suggestion would be to relax formatting rules to allow for two consecutive blank lines as in the attached example.

![Screenshot 2020-10-26 at 08 23 41](https://user-images.githubusercontent.com/6105462/97144397-a659bf00-1764-11eb-9a73-be220df676f0.png)

Reply:
Provisionally removed from this PR.

Reply:
@knoan fwiw I put this up for discussion and seeing if there are some best practices we can agree upon on the rdf4j-dev mailinglist, feel free to chime in there. For now I'm fine with just some developer discretion, as in: feel free to use what you think makes the code easier to read / maintain, as long as nobody objects you're good :)